src/content/plugins/NoEmitOnErrorsPlugin.md

```diff
@@ -0,0 +1,18 @@
+---
+title: NoEmitOnErrorsPlugin
+contributors:
+  - jeffin
+---
+
+The `NoEmitOnErrorsPlugin` allows you to avoid emiting when there are any errors. Enabled by deafult, you can disable using `optimization.noEmitOnErrors`(/configuration/optimization/#optimizationnoemitonerrors)
```

Comment:
```suggestion
The `NoEmitOnErrorsPlugin` allows you to avoid emitting when there are any errors. Enabled by default, you can disable using `optimization.noEmitOnErrors`(/configuration/optimization/#optimizationnoemitonerrors)
```